server/jetstream_api.go

```diff
@@ -863,7 +863,7 @@ func (js *jetStream) apiDispatch(sub *subscription, c *client, acc *Account, sub
 	// header from the msg body. No other references are needed.
 	// Check pending and warn if getting backed up.
 	const warnThresh = 128
-	pending := s.jsAPIRoutedReqs.push(&jsAPIRoutedReq{jsub, sub, acc, subject, reply, copyBytes(rmsg), c.pa})
+	pending, _ := s.jsAPIRoutedReqs.push(&jsAPIRoutedReq{jsub, sub, acc, subject, reply, copyBytes(rmsg), c.pa})
```

Comment:
Ah I see we do use it here and for warning, so limit would not help us here.

Reply:
Yep, this is the only place where the size is returned & used.

Reply:
Well still could be ok, if err then grab len() for the log warning since should not happen too often etc.. WDYT?

Reply:
Left the return value here, since otherwise we need to take the lock again to check the length. Think it's better to avoid lock contention for now than optimise against a return val at this point.